src/main/java/net/fabricmc/loader/api/metadata/ModDependency.java

```diff
@@ -33,4 +33,11 @@
 	 * @param version the version to check
 	 */
 	boolean matches(Version version);
+
+	/**
+	 * Converts this dependency's version range to a user-friendly text representation.
+	 *
+	 * @return a user-friendly representation of this dependency's version range
+	 */
+	String getFriendlyVersionString();
```

Comment:
I somewhat feel like exposing this is in api may not be a great idea for future translation support if someone writes that into loader.

This makes supporting translations in loader very hard as it does not expose the language aspect of these friendly version strings.

Maybe until we decide on a translation system, this should be on a sort of `AbstractModDependency`

Reply:
I could possibly return a `Pair<VersionComparisonOperationEnum, String>` or something like that here. Would also help with future features...

Reply:
Yeah maybe a sort of `VersionComparisonType`.

Reply:
`AbstractModDependency` and `VersionRange` are now things, please check 252d8f2